src/content/structured/get-started/figma.mdx

```diff
@@ -52,6 +45,64 @@ The UI Kit is turned on by default for all projects within our Figma organisatio
   />
 </DoubleDoDontCaution>
 
-## How to use the Figma UI Kit components
-
-Find the details on the internal system to get started with Figma.
+## Not part of the Intelligence Community?
+
+<p>
+  We've published our{" "}
+  <ic-link
+    target="_blank"
+    href="https://www.figma.com/community/file/1283021660431259186"
+    rel="noreferer noopener nofollow"
+  >
+    Figma UI Kit
+  </ic-link>{" "}
+  to the Figma Community so that everyone can use our resources.
+</p>
+<p>
+  You can make a copy of it in your local files and use the components in your
+  designs. This is a static version and you won't automatically receive
+  component updates. Check Figma Community regularly for new updates and
+  versions.
+</p>
+
+<ol>
+  <li>
+    Navigate to the{" "}
+    <ic-link
+      target="_blank"
+      href="https://www.figma.com/community/file/1283021660431259186"
+      rel="noreferer noopener nofollow"
+    >
+      Intelligence Community UI Kit on Figma Community
+    </ic-link>
+    .
+  </li>
+  <li>
+    Duplicate the library into your drafts and connect your project files.
+  </li>
+  <li>
+    Use the{" "}
+    <ic-link
+      target="_blank"
+      href="https://help.figma.com/hc/en-us/articles/360039150413-Swap-components-and-instances"
+      rel="noreferer noopener nofollow"
+    >
+      swap libraries
+    </ic-link>{" "}
+    function in Figma to quickly swap your components from one version to the
+    next.
+  </li>
+</ol>
+<p>
+  If you want to request a new feature or if you notice any issues, you can{" "}
```

Comment:
If we are talking commas, then same problem exists here. 

Not sure what the comma is separating here...

Reply:
It's either:

'If you want to request a new feature or if you notice any issues you can raise it on github issues.'

or

'If you want to request a new feature, or if you notice any issues, you can raise it on github issues.'